### PR TITLE
RHDEVDOCS-5369-cp: Pipelines as Code, Logging Configuration

### PR DESCRIPTION
**Pull request by eromanova97**
This is a manual cherry-pick of #63525 into the standalone.
• **Pipeline version for cherry-picking**: pipelines-docs-1.12
Preview: https://65303--docspreview.netlify.app/openshift-pipelines/latest/pac/pac-command-reference#configuring-pipelines-as-code-logging_pac-command-reference

### CODE REVIEW

**Comment by mramendi** [comment]
/label merge-review-needed

Note: SME. QE, and peer review is already done in #63525 ; only a merge revie and merging remain. The placement of the web UI in CLI reference happens because of a pre-existing absence of CLI information in the logging assembly, which we had no time to fix 1.12 and we will target a fix for 1.13: https://issues.redhat.com/browse/RHDEVDOCS-5636 . I am creating this label as I am the lea writer on Pipelines and this was pending my review.

File: modules/op-configuring-pipelines-as-code-logging.adoc [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                      "durationEncoder": "",
+                      "callerEncoder": ""
+                    }
+                  }
+----
+<1> The log level for the `pipelines-as-code-watcher` component.
+<2> The log level for the `pipelines-as-code-webhook` component.
+<3> The log level for the `pipelines-as-code-controller` component.
+
+. Optional: Create custom logging config map for the {pac} components by changing the `.env.value` for each component under the `.options.deployments` field. The example below shows the configuration with the custom config map called `custom-pac-config-logging`.
```
Comment (mburke5678) [inline modules/op-configuring-pipelines-as-code-logging.adoc:77]:
```suggestion
. Optional: Create a/the custom logging config map for the {pac} components by changing the `.env.value` for each component under the `.options.deployments` field. The example below shows the configuration with the custom config map called `custom-pac-config-logging`.
```

File: modules/op-configuring-pipelines-as-code-logging.adoc [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+. Click the *Instances* tab.
+
+. Click the *config* instance to view the `TektonConfig` CR details.
+
+. Click the *YAML* tab.
+
+. Edit the `loglevel.` fields under the `.options.configMaps.pac-config-logging.data` parameter based on your requirements.
++
+.Example of `TektonConfig` CR with {pac} log level fields set to `warn`
```
Comment (mburke5678) [inline modules/op-configuring-pipelines-as-code-logging.adoc:29]:
```suggestion
.Example of a `TektonConfig` CR with the {pac} log level fields set to `warn`
```

File: modules/op-configuring-pipelines-as-code-logging.adoc [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                    }
+                  }
+----
+<1> The log level for the `pipelines-as-code-watcher` component.
+<2> The log level for the `pipelines-as-code-webhook` component.
+<3> The log level for the `pipelines-as-code-controller` component.
+
+. Optional: Create custom logging config map for the {pac} components by changing the `.env.value` for each component under the `.options.deployments` field. The example below shows the configuration with the custom config map called `custom-pac-config-logging`.
++
+.Example of `TektonConfig` CR with {pac} custom logging config map
```
Comment (mburke5678) [inline modules/op-configuring-pipelines-as-code-logging.adoc:79]:
```suggestion
.Example `TektonConfig` CR with {pac} custom logging config map
```

**Comment by mburke5678** [comment]
@ekristova @mramendi Is this PR waiting for GA like 63525?

**Comment by mramendi** [comment]
@mburke5678 things happened :) 63525 will not be merged, because Pipelines documentation has now moved to a standalone branch. So all PRs that were not yet merged have been cherry-picked into the new branches - pipeline-docs is the "main" for Pipelines, while pipelines-docs-1.12 is the version branch. 

Things can be merged anytime with the new system, we just publish (or more exactly "make user-visible") the new version on GA date.

Regarding this PR, Eliska has a public holiday on Thursday, while GA is on Thursday too. I can't push into her branch. Eliska might log in in the morning and do the changes; if she does not, I will create a new PR in the afternoon with the suggested changes in it and request merge review again.

**Comment by ekristova** [comment]
Requested changes are done :)

**Pull request merged**